CONTRIBUTING.md

```diff
@@ -4,7 +4,9 @@ Welcome to the Nipype repository! We're excited you're here and want to contribu
 
 These guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know by opening an [issue][link_issues]!
 
-Before you start you'll need to set up a free [GitHub][link_github] account and sign in. Here are some [instructions][link_signupinstructions].
+Before you start you'll need to set up a free [GitHub][link_github] account and sign in, here are some [instructions][link_signupinstructions].
```

Comment:
The comma you added doesn't really work. You can use a period as before, or switch to a semicolon (`;`) and keep the lowercase "here".